src/bionics.cpp

```diff
@@ -2317,6 +2319,54 @@ bool Character::uninstall_bionic( const bionic &target_cbm, monster &installer,
     return false;
 }
 
+ret_val<bool> Character::is_installable( const item_location &loc, const bool by_autodoc ) const
+{
+    const item *it = loc.get_item();
+    const itype *itemtype = it->type;
+    const bionic_id &bid = itemtype->bionic->id;
+
+    const auto has_trait_lambda = [this]( const trait_id & candidate ) {
+        return has_trait( candidate );
+    };
+
+    if( it->has_flag( flag_FILTHY ) ) {
+        // NOLINTNEXTLINE(cata-text-style): single space after the period for symmetry
+        const std::string msg = by_autodoc ? _( "/!\\ CBM is highly contaminated. /!\\" ) :
+                                _( "CBM is filthy." );
+        return ret_val<bool>::make_failure( msg );
+    } else if( it->has_flag( flag_NO_STERILE ) ) {
+        // NOLINTNEXTLINE(cata-text-style): single space after the period for symmetry
+        const std::string msg = by_autodoc ?
+                                _( "/!\\ CBM is not sterile. /!\\ Please use autoclave to sterilize." ) :
```

Comment:
```suggestion
                                _( "/!\\ CBM is not sterile.  /!\\ Please use autoclave to sterilize." ) :
```

Reply:
The string is correct, but the `nolint` comment is not at the correct place.